models/gif_pop_psc_exp.cpp

```diff
@@ -553,37 +550,37 @@ nest::gif_pop_psc_exp::update( Time const& origin,
     S_.y0_ = B_.currents_.get_value( lag );
 
     // begin procedure update population, Fig. 12
-    double W_ = 0, X_ = 0, Y_ = 0, Z_ = 0; // line 2
+    double W_ = 0, X_ = 0, Y_ = 0, Z_ = 0; // line 2 of [1]
     S_.theta_hat_ = P_.V_T_star_;          // line 2, initialize theta
 
-    S_.V_m_ = ( S_.V_m_ - P_.E_L_ ) * V_.P22_ + h_tot_; // line 3
+    S_.V_m_ = ( S_.V_m_ - P_.E_L_ ) * V_.P22_ + h_tot_; // line 3 of [1]
 
     // compute free adaptation state
-    for ( uint j = 0; j < P_.tau_sfa_.size(); j++ ) // line 4
+    for ( uint j = 0; j < P_.tau_sfa_.size(); ++j ) // line 4 of [1]
     {
-      V_.g_[ j ] = V_.g_[ j ] * V_.Q30_[ j ]
-        + ( 1. - V_.Q30_[ j ] ) * V_.n_[ V_.k0_ ]
-          / ( static_cast< double >( P_.N_ ) * V_.h_ );           // line 5
+      double g_j_tmp_ = ( 1. - V_.Q30_[ j ] ) * V_.n_[ V_.k0_ ]
```

Comment:
As above: declare local helper variables `const` where possible and avoid the trailing `_`.